service-workers/service-worker/META.yml

```diff
@@ -216,3 +216,27 @@ links:
       url: https://bugs.chromium.org/p/chromium/issues/detail?id=915140
       results:
         - test: fetch-request-css-base-url.https.html
+    - product: chrome
+      url: https://crbug.com/701234
+      results:
+        - test: clients-matchall-order.https.html
+    - product: chrome
+      url: https://crbug.com/906991
+      results:
+        - test: dedicated-worker-service-worker-interception.https.html
+    - product: chrome
+      url: https://crbug.com/900700
+      results:
+        - test: fetch-event-worker-timing-frame.tentative.https.html
+    - product: chrome
+      url: https://crbug.com/1052642
+      results:
+        - test: unregister-immediately-during-extendable-events.https.html
+    - product: chrome
+      url: https://crbug.com/832071
```

Comment:
Bug was originally about the test having unstable output, but has pivoted to be about fixing the Chrome failure.